pkg/lib/pb/model/protos/models.proto

```diff
@@ -1186,12 +1172,16 @@ message Membership {
     }
     
     enum PaymentMethod {
-        MethodCard = 0; // Stripe
```

Comment:
maybe it's worth adding 'None' as the default value, for a situation when you can buy this Tier?